src/services/SendEmailProcessor.ts

```diff
@@ -32,13 +32,13 @@ export class SendEmailProcessor {
     	return SendEmailProcessor.instance;
 	}
 
-	async processRequest(eventBody: any): Promise<EmailResponse> {
+	async processRequest(eventBody: any): Promise<EmailResponse | undefined> {
```

Comment:
can it return undefined? 🤔

Reply:
When it throws an error, in that case it will undefined.